settings.gradle

```diff
@@ -2,5 +2,5 @@ rootProject.name = 'fw-monitoring-examples'
 include 'ktor-monitoring'
 include 'spring-boot-monitoring'
 include 'micronaut-monitoring'
+include 'micronaut-monitoring-kt'
```

Comment:
is possible to follow proposed naming <framework>-<language>-<purpose>, example _micronaut-kt-monitoring_ ?